tests/rt.conf

```diff
@@ -116,194 +116,194 @@ RUN | regional_ifi_2threads                             | + acorn
+COMPILE | hafsw | intel | -DAPP=HAFSW -DMOVING_NEST=ON -DCCPP_SUITES=FV3_HAFS_v1_gfdlmp_tedmf,FV3_HAFS_v1_gfdlmp_tedmf_nonsst,FV3_HAFS_v1_thompson_tedmf_gfdlsf -D32BIT=ON |  | fv3 |
 RUN | hafs_regional_atm                                 |                            | baseline |
 RUN | hafs_regional_atm_thompson_gfdlsf                 |                            | baseline |
 RUN | hafs_regional_atm_ocn                             |                            | baseline |
 RUN | hafs_regional_atm_wav                             |                            | baseline |
-RUN | hafs_regional_atm_ocn_wav                         |                            | baseline |
-RUN | hafs_regional_1nest_atm                           | - jet s4                   | baseline |
-RUN | hafs_regional_1nest_atm_qr                        | - jet s4 cheyenne          |          |
-RUN | hafs_regional_telescopic_2nests_atm               | - jet s4                   | baseline |
-RUN | hafs_global_1nest_atm                             | - jet s4                   | baseline |
-RUN | hafs_global_1nest_atm_qr                          | - jet s4 cheyenne          |          |
-RUN | hafs_global_multiple_4nests_atm                   | - jet s4                   | baseline |
-RUN | hafs_global_multiple_4nests_atm_qr                | - jet s4 cheyenne          |          |
-RUN | hafs_regional_specified_moving_1nest_atm          | - jet s4                   | baseline |
-RUN | hafs_regional_storm_following_1nest_atm           | - jet s4                   | baseline |
-RUN | hafs_regional_storm_following_1nest_atm_qr        | - jet s4 cheyenne          |          |
-RUN | hafs_regional_storm_following_1nest_atm_ocn       | - jet s4                   | baseline |
-RUN | hafs_global_storm_following_1nest_atm             | - jet s4                   | baseline |
-
-COMPILE | hafsw_debug | intel | -DAPP=HAFSW -DMOVING_NEST=ON -DCCPP_SUITES=FV3_HAFS_v1_gfdlmp_tedmf_nonsst -D32BIT=ON -DDEBUG=ON | - jet s4 | fv3 |
-RUN | hafs_regional_storm_following_1nest_atm_ocn_debug | - jet s4                   | baseline |
-
-COMPILE | hafsw_faster | intel | -DAPP=HAFSW -DMOVING_NEST=ON -DCCPP_SUITES=FV3_HAFS_v1_gfdlmp_tedmf_nonsst -D32BIT=ON -DFASTER=ON | -jet s4 | fv3 |
-RUN | hafs_regional_storm_following_1nest_atm_ocn_wav   | - jet s4                   | baseline |
-
-COMPILE | hafs_all | intel | -DAPP=HAFS-ALL -DCCPP_SUITES=FV3_HAFS_v1_gfdlmp_tedmf,FV3_HAFS_v1_gfdlmp_tedmf_nonsst -D32BIT=ON | - wcoss2 | fv3 |
-RUN | hafs_regional_docn                                | - wcoss2                   | baseline |
-RUN | hafs_regional_docn_oisst                          | - wcoss2                   | baseline |
-RUN | hafs_regional_datm_cdeps                          | - wcoss2 -jet s4           | baseline |
+RUN | hafs_regional_atm_ocn_wav                         | - noaacloud                | baseline |
```

Comment:
Could you please re-align the baseline columns where needed?

Reply:
Columns should be straightened out now.